.env

```diff
@@ -0,0 +1,9 @@
+NEXT_PUBLIC_API_URL="http://127.0.0.1:8000/api/"
+GOOGLE_CLIENT_ID=185796748522-q2t69jsidgopsqji9b7eujgqk945up7j.apps.googleusercontent.com
```

Comment:
Never submit secrets to git. @JoaoCVerissimo when you're available help out here